CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.25.2-dev
+* Bug - Update pull image retry to longer internal to mitigate being throttled by ecr
```

Comment:
* `interval`? 
ecr in uppercase 
Could you please add the PR link as well?

Reply:
updated 👍